### ISSUE TITLE
Test failing due to latex

### ISSUE THREAD

**Issue by mykelk** [JuliaPOMDP/POMDPGallery.jl#9]
It looks like we have an issue with testing on the travis build due to missing `luatex85.sty`. Should we not have TikzPictures as part of the testing? Or should we add the dependency to have a proper install?

**Comment by zsunberg** [comment]
Yes, that seems like a good idea - we also could just switch to a Compose-based approach. If it is for the lasertag problem. I have a compose-based visualization for a similar problem here: https://github.com/zsunberg/DMUStudent.jl/blob/1db6d1de32f81df43cb1ee36ea1d4263b5c1d708/src/HW6.jl#L199

Adding TikzPictures is probably the easiest fix.
